doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
```

Comment:
and please add your notebook to the list here https://github.com/ICB-DCM/pyPESTO/blob/main/test/run_notebook.sh#L17 (does it run in less than say a minute?) so that it is run regularly with the tests.

Reply:
(if it has special requirements, make sure those are installed)